Sources/Subs.php

```diff
@@ -1750,9 +1747,9 @@ function parse_bbc($message, $smileys = true, $cache_id = '', $parse_tags = arra
 					'alt' => array('optional' => true),
 					'title' => array('optional' => true),
 					'width' => array('optional' => true, 'value' => ' width="$1"', 'match' => '(\d+)'),
-					'height' => array('optional' => true, 'value' => ' height="$1"', 'match' => '(\d+)'),
+					'height' => array('optional' => true, 'value' => '', 'match' => '(\d+)'),
 				),
-				'content' => '<img src="$1" alt="{alt}" title="{title}"{width}{height} class="bbc_img resized" loading="lazy">',
+				'content' => '<img src="$1" alt="{alt}" title="{title}"{width} class="bbc_img resized" loading="lazy">',
```

Comment:
google and user didn't like lazy load without height.

Reply:
Fixed